.obs/seedimage/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: Apache-2.0
+# Define the names/tags of the container
+#!BuildTag: rancher/seedimage-builder/5.3:latest
+#!BuildTag: rancher/seedimage-builder/5.3:%OPERATOR_VERSION%
+#!BuildTag: rancher/seedimage-builder/5.3:%OPERATOR_VERSION%-%RELEASE%
+#
+
+FROM suse/sle15:15.4 as BASE
+
+RUN mkdir /installroot
+RUN  mkdir -p /installroot/etc/products.d && \
+    cp /etc/products.d/baseproduct /installroot/etc/products.d/ && \
```

Comment:
Whats the purpose of this? Why is this actually needed?

I am not sure we can actually do that,  this belongs to a release package.

Reply:
Right, this is a leftover which we should not need here... removed.